exercises/pig-latin/.meta/src/reference/java/PigLatinTranslator.java

```diff
@@ -21,11 +21,17 @@ String translate(String sentence) {
                 .collect(Collectors.joining(" "));
     }
 
+    private static int yLocation;
```

Comment:
Having a private variable which is set in one method and used in another like this makes the code a bit confusing to read. If you need to use this straight after having called `hasYAfterConsonantCluster` then you could just make the method return the yLocation (and rename it something like `getLocationOfYAfterConsonantCluster`)? And it can return -1 if there isn't one. What do you think? 🙂

Reply:
Great idea. I'll take care of it. Thanks!